sources/engine/Stride/Graphics/Image.cs

```diff
@@ -136,6 +136,28 @@ public sealed class Image : IDisposable
         /// </summary>
         public ImageDescription Description;
 
+        /// <summary>
+        /// Converts the format of the description and the pixel buffers to SRgb.
+        /// </summary>
+        public void ToSRgb()
```

Comment:
It sounds like it returns a SRGB version of this image. Maybe this should be renamed `ConvertToSRgb` to show it's a mutable action?